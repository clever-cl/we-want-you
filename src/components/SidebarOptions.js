import React from 'react'
import "../styles/SidebarOptions.css"
const SidebarOptions = ({title, Icon}) => {
    return (
      <div className="sidebarOption">
      {Icon && <Icon className="sidebarOption__icon" />}
      {Icon ? <h4>{title}</h4> : <p>{title}</p>}
    </div>
    )
}

export default SidebarOptions
