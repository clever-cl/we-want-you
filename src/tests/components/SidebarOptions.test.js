import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme';
import SidebarOptions from '../../components/SidebarOptions';

describe('Test in <SidebarOptions />', () => {
  test('should show component correct', () => {
    const wrapper = shallow(<SidebarOptions />)
    expect(wrapper).toMatchSnapshot()
  })


})