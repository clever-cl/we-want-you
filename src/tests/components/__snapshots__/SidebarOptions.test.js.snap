// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <SidebarOptions /> should show component correct 1`] = `
<div
  className="sidebarOption"
>
  <p />
</div>
`;
